evalml/automl/automl_search.py

```diff
@@ -383,18 +385,24 @@ def search(self, X, y, data_checks="auto", feature_types=None, show_iteration_pl
             except NameError:
                 show_iteration_plot = False
 
-        # make everything pandas objects
-        if not isinstance(X, pd.DataFrame):
-            X = pd.DataFrame(X)
+        # make everything ww objects
+        if not isinstance(X, ww.DataTable):
+            if isinstance(X, np.ndarray):
+                X = pd.DataFrame(X)
+            X = ww.DataTable(X)
 
-        if not isinstance(y, pd.Series):
-            y = pd.Series(y)
+        if not isinstance(y, ww.DataColumn):
+            if isinstance(y, np.ndarray):
+                y = pd.Series(y)
+            y = ww.DataColumn(y)
+
+        X = _convert_nullable_types_wrapper(X.to_pandas())
```

Comment:
This breaks AutoML on the fraud dataset because woodwork has a `NaturalLanguage` logical type that gets converted to a pandas `string` dtype. The OHE will not encode that column (we check for `object` or `categorical` pandas dtypes). I think we should convert `string` to `bool` in `_convert_nullable_types_wrapper`. We can do a better fix in #1289?

![image](https://user-images.githubusercontent.com/41651716/96601879-6f952c00-12c0-11eb-8ca6-1a84c6d24cb5.png)

Reply:
Yup, going to update to map StringDtype to object. Thanks for catching this!